install/install.sh

```diff
@@ -217,8 +231,24 @@ docker create \
     -e BLUEOS_CONFIG_PATH=$HOME/.config/blueos \
     $BLUEOS_BOOTSTRAP
 
-# add docker entry to rc.local
-sed -i "\%^exit 0%idocker start blueos-bootstrap" /etc/rc.local || echo "Failed to add docker start on rc.local, BlueOS will not start on boot!"
+if [ "$OS" = "Raspbian" ]; then
+    # add docker entry to rc.local
+    sed -i "\%^exit 0%idocker start blueos-bootstrap" /etc/rc.local || echo "Failed to add docker start on rc.local, BlueOS will not start on boot!"
+
+elif [ "$OS" = "Ubuntu" ]; then
+    # Create a systemd service file
+    echo "[Unit]
+    Description=Start Docker container blueos-bootstrap at system start
+
+    [Service]
+    ExecStart=/usr/bin/docker start blueos-bootstrap
+
+    [Install]
+    WantedBy=multi-user.target" > /etc/systemd/system/blueos-bootstrap.service
+
+    systemctl daemon-reload
+    systemctl enable blueos-bootstrap.service
```

Comment:
Why docker start does not work with ubuntu like raspbian ? We rely on docker to start with the system, and that as consequence will start bootstrap.

Reply:
Ubuntu does not appear to use rc.local, so to get the bootstrap docker to start with the system, you need to use some other method like a systemd service, as I did here.

Reply:
Oh, thanks!